packages/fluentui/react-northstar/src/components/Menu/MenuItem.tsx

```diff
@@ -427,7 +465,7 @@ export const MenuItem = compose<'a', MenuItemProps, MenuItemStylesProps, {}, {}>
               targetRef={itemRef}
               {...getPopperPropsFromShorthand(menu)}
             >
-              {Menu.create(menu, {
+              {createShorthand(parentProps.__parentComponent || Menu, menu, {
```

Comment:
If we want to support custom components for the submenus, we may use here `composeOptions.slots.menu` as a first option... But to be honest, for now would keep it simple